msmarco-v2-vector/challenges/default.json

```diff
@@ -55,26 +55,166 @@
     {
       "name": "standalone-search-knn-10-100-single-client",
       "operation": "knn-search-10-100",
-      "warmup-iterations": 100,
+      "warmup-iterations": 1000,
```

Comment:
I am curious for the reason of changing the warmup config?

Reply:
Variations in the result so mainly to stabilise the report.